pkg/render/fluentd.go

```diff
@@ -154,7 +220,10 @@ func (c *fluentdComponent) Objects() ([]runtime.Object, []runtime.Object) {
 			c.eksLogForwarderSecret(),
 			c.eksLogForwarderDeployment())
 	}
-	if c.installation.KubernetesProvider != operatorv1.ProviderOpenShift {
+
+	// Windows does not support allowedHostPaths yet.
```

Comment:
I think this comment is supposed to be about PSPs

Reply:
Added a clarification